frame/staking/src/lib.rs

```diff
@@ -467,7 +466,7 @@ pub struct StakingLedger<T: Config> {
 	pub unlocking: BoundedVec<UnlockChunk<BalanceOf<T>>, T::MaxUnlockingChunks>,
 	/// List of eras for which the stakers behind a validator have claimed rewards. Only updated
 	/// for validators.
-	pub claimed_rewards: BoundedVec<EraIndex, T::HistoryDepth>,
+	pub legacy_claimed_rewards: BoundedVec<EraIndex, T::HistoryDepth>,
```

Comment:
Perhaps we should add a comment here explaining why this is legacy and link to an issue that keeps track of removing it (I assume that it will eventually be removed, otherwise perhaps the name is misleading).